src/main/plugin/iso19139.ca.HNAP/templates/hnaplevel1-english.xml

```diff
@@ -372,9 +370,8 @@
                               <gco:Date>2016-07-04</gco:Date>
                            </gmd:date>
                            <gmd:dateType>
-                              <gmd:CI_DateTypeCode codeList="http://nap.geogratis.gc.ca/metadata/register/napMetadataRegister.xml#IC_87"
+                              <gmd:CI_DateTypeCode codeList="https://schemas.metadata.geo.ca/register/napMetadataRegister.xml#IC_87"
                                                    codeListValue="RI_366">creation; création</gmd:CI_DateTypeCode>
-                           </gmd:dateType>
```

Comment:
Why is gmd:dateType ending element removed?

Reply:
conflict resolution whitespace change again.